asian-recipes.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Asian Chicken Recipes</title>
+  </head>
+  <body>
+    <center>
+      <h1>Asian style chicken</h1>
+
+      <a
+        href="/Users/mz/Documents/bootcamp/m1/day1/recipe-site-bootcamp/kung-pao-chicken.html"
+        ><h2>Kung pao chicken</h2></a
+      >
```

Comment:
Your code formatter does not seem to be setup correctly. Let's check that before or during the next lesson